setup.py

```diff
@@ -23,5 +23,6 @@
         'pefile',
         'sortedcontainers>=2.0',
         'minidump==0.0.10',
+        'pyxbe==0.0.2',
```

Comment:
Why do we need to pin to a specific version of pyxbe?

Reply:
The library is still pretty new. Though I don't expect to, if I make any changes to the API, I don't want it to break CLE. Seeing as 0.0.2 is stable enough, I think it's good to pin it here and we can bump the revision up as I add features or bug fixes to pyxbe.